src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEngine.Unix.cs

```diff
@@ -13,6 +13,8 @@ namespace System.Net.Sockets
 {
     internal sealed unsafe class SocketAsyncEngine
     {
+        internal static readonly bool InlineSocketContinuations = Environment.GetEnvironmentVariable("DOTNET_INLINE_SOCKET_CONTINUATIONS") == "1";
```

Comment:
it would be great to add some comment here that explains what this flag does

btw. I like the  fact that the env var name does  not start with `COMPlus_` ;)